### PR TITLE
Part 1: Study burst refactor

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Requires: https://github.com/Sage-Bionetworks/BridgeApp-Apple-SDK/pull/54

This adds local resources for the Triggers, Symptoms, and Medication.

### CODE REVIEW

File: mPower2/mPower2/Medication.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+              "identifier": "Stalevo",
+              "detail": "(Levodopa / Carbidopa and Entacapone)"
+              },
+              {
+              "sectionIdentifier": "Catechol-O-methyltransferase inhibitors",
+              "identifier": "Tasmar",
+              "detail": "(Tolcapone)"
+              },
+              {
+              "sectionIdentifier": "Dopamine promotor",
```
Comment (Erin-Mounts) [inline mPower2/mPower2/Medication.json]: promotor --> promoter (here and following)

File: mPower2/mPower2/Medication.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Sinemet CR",
+              "detail": "(Carbidopa / Levodopa)"
+              },
+              {
+              "sectionIdentifier": "Dopamine replacement therapy",
+              "identifier": "Stalevo",
+              "detail": "(Carbidopa / Levodopa)"
```
Comment (Erin-Mounts) [inline mPower2/mPower2/Medication.json]: Stalevo is listed in a different section above with (Levodopa / Carbidopa and Entacapone)

File: mPower2/mPower2/Triggers.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
+{
+    "identifier": "Triggers",
```
Comment (Erin-Mounts) [inline mPower2/mPower2/Triggers.json]: Weren't we going to change the name of Triggers to something else? seems like there was a JIRA for that (but maybe this is just a copypaste or moving an existing file and the Triggers rename will happen in a separate PR, in which case just ignore this)

File: mPower2/mPower2/study-burst-task.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "identifier": "study-burst-task",
+    "type": "studyBurst",
+    "numberOfDays": 14,
+    "activeTaskGroupIdentifier": "Measuring",
+    "competionTasks" : [
```
Comment (Erin-Mounts) [inline mPower2/mPower2/study-burst-task.json]: comp**l**etionTasks

File: mPower2/mPower2/Medication.json (hunk 2 above)
Comment (syoung-smallwisdom) [reply mPower2/mPower2/Medication.json]: Yes, I know. ;)

File: mPower2/mPower2/Triggers.json (hunk 3 above)
Comment (syoung-smallwisdom) [reply mPower2/mPower2/Triggers.json]: Um... I don't remember that, though yes, this is a copy/paste.

File: mPower2/mPower2/study-burst-task.json (hunk 4 above)
Comment (syoung-smallwisdom) [reply mPower2/mPower2/study-burst-task.json]: This file is WIP. Added so that the project build won't fail, but should be ignored for now.

**Pull request merged**